core/src/main/java/org/apache/iceberg/hadoop/HadoopTableOperations.java

```diff
@@ -154,21 +155,33 @@ public void commit(TableMetadata base, TableMetadata metadata) {
     Path tempMetadataFile = metadataPath(UUID.randomUUID().toString() + fileExtension);
     TableMetadataParser.write(metadata, io().newOutputFile(tempMetadataFile.toString()));
 
+    // todo:What should we do if version variable overflows?
```

Comment:
Make an Issue instead of a todo, I still don't think this is a reasonable thing to worry about since even with 1 commit a second this would be 60~ years in the future

Reply:
@RussellSpitzer 
I agree, I was just trying to keep a to-do list at first. It can indeed be deleted.